fastats/linear_algebra/ols.py

```diff
@@ -204,6 +204,33 @@ def t_statistic(A, b):
     return betas / se
 
 
+def drop_missing(A, b):
+    """
+    Returns a filtration of A (features) and b (targets) where all
+    values are not NaN, with the intention that OLS can then be
+    performed on dense / 'complete' observations.
+
+    This is analogous to the statsmodels missing='drop' mechanism.
+    """
+    m = A.shape[0]
+    assert m == len(b)
+
+    A_bar = np.empty_like(A)
+    b_bar = np.empty_like(b)
+
+    ctr = 0
+
+    for i in range(m):
+        feature_row = A[i, :]
+        target = b[i]
+        if np.all(~np.isnan(feature_row)) and ~np.isnan(target):
+            A_bar[ctr, :] = feature_row
+            b_bar[ctr] = target
+            ctr += 1
+
+    return A_bar[:ctr, :], b_bar[:ctr]
```

Comment:
Would be nice to have another function or parameter to return the indices that have been dropped - this way pandas indices etc can be re-constructed.

Reply:
Yep, nice one